Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -0,0 +1,269 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Collections.Generic;
+using System.Linq;
+using Editor.CoreAPI;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace AmazonGameLift.Editor
+{
+    internal class AwsUserProfilesPage
+    {
+        public List<TextField> AccountDetailTextFields = new();
+        
+        public readonly AwsCredentialsUpdate UpdateModel;
+        public readonly AwsCredentialsCreation CreationModel;
```

Comment:
Please rename this to `awsCredentialsUpdateModel` and `awsCredentialsCreateModel`

Reply:
Do you want the class or the variable changed?

Reply:
I wouldn't update the class name since it's a bigger change, maybe just use the variable name to be `awsCredentialsUpdate`. For future reference: We typically name the variable name very similar if not exactly the same as the class name, unless there's a better contextual name available which can explain what it's meant for. In this case since I can't think of a contextual name, I just use the class name itself.

Reply:
Renamed